tools/packages.yaml

```diff
@@ -8136,6 +8136,11 @@ url: http://kernel.ubuntu.com/~cking/tarballs/smemstat
 activity: medium
 ---
 kind: url
+name: smplayer
+url: https://github.com/smplayer-dev/smplayer/releases
```

Comment:
I thought we were using `version.rb` instead of `tools/packages.yaml`.

Reply:
`version.rb` is still not usable for me.  Some package lookups are not the same project and many times I get no results at all.  I did have it working quite well at one point but then SOMEONE decided to change it and make it work not so well after that. 😛

Reply:
![image](https://github.com/chromebrew/chromebrew/assets/64618338/ef528e79-810a-4155-ae5f-7d810f6b66c8)

I've tested it, and it works for smplayer-- If you still want to add it to `tools/packages.yaml` go ahead i guess

Reply:
I think we need to use multiple sources for package versions.  The way to make it more comprehensive would be to use another source if no results are found with the first repository.